src/Host.h

```diff
@@ -438,6 +439,7 @@ class Host : public QObject
     void setCommandLineHistorySaveSize(const int lines);
     bool showIdsInEditor() const { return mShowIDsInEditor; }
     void setShowIdsInEditor(const bool isShown) { mShowIDsInEditor = isShown; if (mpEditorDialog) {mpEditorDialog->showIDLabels(isShown);} }
+    QNetworkAccessManager* getNetworkAccessManager() const { return mpNetworkAccessManager; }
```

Comment:
Also, given that other classes are using this - it might be good to include a check that they are not doing so at the wrong time - so debug builds at least will explode consistently if they do:
```suggestion
    Q_ASSERT_X(mpNetworkAccessManager, "Host::getNetworkAccessManager()", "is not valid at this time, it is a nullptr!");
    QNetworkAccessManager* getNetworkAccessManager() const { return mpNetworkAccessManager; }
```